packages/presentation/lib/screen/home/home_state.dart

```diff
@@ -13,19 +13,22 @@ class HomeState {
   final int? predictedNumber;
   final int? generatedNumber;
   final int attempts;
+  final bool isAttemptsTextAvailable;
   final bool isSubmitButtonActive;
   final GameState gameState;
 
   factory HomeState.init() => HomeState(
         attempts: 0,
         isSubmitButtonActive: false,
         gameState: GameState.inGame,
+        isAttemptsTextAvailable: false,
       );
 
   HomeState copy() => HomeState(
```

Comment:
do not seen, is this method is used anywhere?

Reply:
Sure! It's used for make text "Try again. You have .. attempts" available. 
[packages/presentation/lib/screen/home/home_page.dart] line 110

Reply:
I ask about copy() method =)  As I remember u used copyWith() , but copy() is not

Reply:
You highlight another line. Ok, I I really didn't used this method. I delete it.